Client/sdk/game/CPed.h

```diff
@@ -15,6 +15,7 @@
 #include "Common.h"
 #include "CPhysical.h"
 #include "CWeaponInfo.h"
+#include <vector>
```

Comment:
Is this really needed here? From what I can see `vector` is already included in PCH of both game_sa and deathmatch.

Reply:
If it wasn't needed, I wouldn't add it. Without this I get the error "std namespace element does not have a vector member".

Reply:
Okay. I'm just wondering during the compilation of which project exactly this error was occured?

Reply:
However, it's Visual Studio that goes crazy - Compilation, runs fine but Visual Studio shows this error
![image](https://github.com/multitheftauto/mtasa-blue/assets/26044224/e1154715-3254-4ca8-bfa8-1a23f924b93c)

Reply:
> However, it's Visual Studio that goes crazy - Compilation, runs fine but Visual Studio shows this error ![image](https://private-user-images.githubusercontent.com/26044224/344133484-e1154715-3254-4ca8-bfa8-1a23f924b93c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk1NzUzNjYsIm5iZiI6MTcxOTU3NTA2NiwicGF0aCI6Ii8yNjA0NDIyNC8zNDQxMzM0ODQtZTExNTQ3MTUtMzI1NC00Y2E4LWJmYTgtMWEyM2Y5MjRiOTNjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI4VDExNDQyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxZDM4ZmIxOGU4NjUyMGI3ZjEyNTVkMjYyNmZmNmE2YmJlNjkxODQ0MWI5MGQzMGZjZTM2Y2ZjNDdmNGIyYWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-xBWaTUXjh5ryeAcLFi_Tgp9VIp7Kt7ykrNGr6A1FMs)

then the vector class was not found in the current translation unit